### ISSUE TITLE
Tutorial example is not rendered

### ISSUE THREAD

**Issue by ohad** [vega/ipyvega#354]
I'm following the installation instructions in the README:

```bash
wget https://raw.githubusercontent.com/vega/ipyvega/master/notebooks/cars.json
pip3 install jupyter pandas vega
pip3 install --upgrade notebook
jupyter notebook
```
then follow the tutorial:
```
-------------
from vega import VegaLite
import pandas as pd

df = pd.read_json('cars.json')
df.head()
--------------
vl = VegaLite({
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "mark": "point",
  "encoding": {
    "y": {"type": "quantitative", "field": "Acceleration"},
    "x": {"type": "quantitative", "field": "Horsepower"}
  }
}, df)
print(vl)
vl
```
I expected to get a similar output to the [tutorial](https://github.com/vega/ipyvega/blob/master/screenshot.png), but the figure isn't rendered:

![vega-tutorial](https://user-images.githubusercontent.com/536511/126767138-bdb6b070-04fb-4784-bcf7-eefc0fae4692.png)

I guess I'm missing a configuration step, apologies if this is somehow obvious!

Currently installed packages:
```bash
$ pip3 show jupyter pandas vega notebook
Name: jupyter
Version: 1.0.0
Summary: Jupyter metapackage. Install all the Jupyter components in one go.
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.org
License: BSD
Location: /home/ohad/.local/lib/python3.6/site-packages
Requires: nbconvert, qtconsole, ipywidgets, jupyter-console, ipykernel, notebook
Required-by: vega
---
Name: pandas
Version: 1.1.5
Summary: Powerful data structures for data analysis, time series, and statistics
Home-page: https://pandas.pydata.org
Author: None
Author-email: None
License: BSD
Location: /home/ohad/.local/lib/python3.6/site-packages
Requires: python-dateutil, numpy, pytz
Required-by: vega
---
Name: vega
Version: 3.5.0
Summary: A Jupyter widget for Vega 5 and Vega-Lite 4
Home-page: https://github.com/vega/ipyvega
Author: Dominik Moritz
Author-email: domoritz@gmail.com
License: BSD 3-clause
Location: /home/ohad/.local/lib/python3.6/site-packages
Requires: jupyter, pandas
Required-by: 
---
Name: notebook
Version: 6.4.0
Summary: A web-based notebook environment for interactive computing
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /home/ohad/.local/lib/python3.6/site-packages
Requires: prometheus-client, ipython-genutils, traitlets, jupyter-core, nbformat, nbconvert, jupyter-client, terminado, pyzmq, ipykernel, argon2-cffi, tornado, jinja2, Send2Trash
Required-by: widgetsnbextension, jupyter
```

**Comment by domoritz** [comment]
That should work. Can you try the example notebook in this repo? Can you try to restart the browser? Any errors in the browser console?

**Comment by ohad** [comment]
Thanks for picking this up quickly!

+ [X]  Try the notebook in this repo: Yes, I'm getting the same unexpected behaviour.
+ [ ] Restart the browser: 
+ [X] Any errors in the browser console? I have some, but it's not clear to me they come from this error:
```
tried to detect trackers before rules were loaded background.js:450
    getTrackerData moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:450
    handleRequest moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:21181
    apply self-hosted:2732
    applySafeWithoutClone resource://gre/modules/ExtensionCommon.jsm:626
    fire resource://gre/modules/ExtensionChild.jsm:806
    recvRunListener resource://gre/modules/ExtensionChild.jsm:810
    recvRunListener self-hosted:1175
    _recv resource://gre/modules/ConduitsChild.jsm:78
    receiveMessage resource://gre/modules/ConduitsChild.jsm:184
    run resource://gre/modules/ConduitsChild.jsm:175
    receiveMessage resource://gre/modules/ConduitsChild.jsm:176
    map self-hosted:221
    receiveMessage resource://gre/modules/ConduitsChild.jsm:176
this.trackerList is undefined background.js:524
    findTracker moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:524
    isTracker moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:22300
    truncateReferrer moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:22527
    limitReferrerData moz-extension://25dab8af-a127-4050-8335-23a0a75b30f2/public/js/background.js:18413
    apply self-hosted:2732
    applySafeWithoutClone resource://gre/modules/ExtensionCommon.jsm:626
    fire resource://gre/modules/ExtensionChild.jsm:806
    recvRunListener resource://gre/modules/ExtensionChild.jsm:810
    recvRunListener self-hosted:1175
    _recv resource://gre/modules/ConduitsChild.jsm:78
    receiveMessage resource://gre/modules/ConduitsChild.jsm:184
    run resource://gre/modules/ConduitsChild.jsm:175
    receiveMessage resource://gre/modules/ConduitsChild.jsm:176
    map self-hosted:221
    receiveMessage resource://gre/modules/ConduitsChild.jsm:176
```

**Comment by domoritz** [comment]
I don't know how to help then. I suspect this is another issue with the setup and not a bug in the library.

**Comment by ohad** [comment]
Well thanks for the help! I assume you cannot reproduce this on your end?

**Comment by domoritz** [comment]
Just tried this in a conda environment.

<img width="1167" alt="Screen Shot 2021-07-23 at 11 58 47" src="https://user-images.githubusercontent.com/589034/126828903-77ce7f8e-90e8-4602-a5f5-440210dc28d5.png">

**Comment by ohad** [comment]
OK, thanks! I'll ask around and see if I can reproduce it with pip on another system.

**Comment by ohad** [comment]
I think there's something missing in the `pip` installation instructions.

I created a new Dockerfile:
```dockerfile
FROM ubuntu:latest
RUN apt-get update && apt-get -y update
RUN apt-get install -y build-essential python3.6 python3-pip python3-dev firefox wget
RUN pip3 -q install pip --upgrade
RUN pip3 -q install jupyter pandas vega
RUN pip3 -q install --upgrade notebook
RUN wget https://raw.githubusercontent.com/vega/ipyvega/master/notebooks/cars.json
```
then launched it with gui enabled:
```bash
docker build -t issue-354 .
xhost +   # Don't forget to xhost 
docker run -it --env="DISPLAY" --net=host issue-354 # We should now be inside the container
jupyter notebook --allow-root # firefox should now launch with a notebook
```
This reproduces the behaviour I described, the image doesn't appear.

**Comment by domoritz** [comment]
Does it work if you run `jupyter nbextension install --sys-prefix --py vega` before starting the kernel?

**Comment by ohad** [comment]
Thanks, esp. for the quick replies! I confirm that now works (in the container and on my machine).
I was thrown off by the qualification: '# not needed in notebook >= 5.3'. 
Maybe we ought to do it always anyway?
